src/Data/Model.js

```diff
@@ -316,6 +316,9 @@ class Model extends Observable
 
 				// Is the value computed?
 				if (value instanceof ComputedProperty) return false;
+				
+				// Is the value the 'is' observable
+				if (value instanceof Observable && key == 'is') return false;
```

Comment:
Hoe komt die 'is' hier uberhaupt in de attributes van dat model? Even hierboven wordt die `attr` array gevuld, maar daar hoort ie al niet in toch?